isce2_topsapp/delivery_prep.py

```diff
@@ -115,26 +124,30 @@ def format_metadata(nc_path: Path,
     b_perp = read_baseline_perp(nc_path).mean()
 
     metadata = {}
-    metadata.update({"ogr_bbox": geojson['coordinates'],
+    # get 4 corners of bounding box of the geometry; default is 5 returning
+    # to start point
+    ogr_bbox = all_metadata['intersection_geo'].envelope.exterior.coords[:4]
+    metadata.update({"ogr_bbox": ogr_bbox,
                      "reference_scenes": all_metadata['reference_scenes'],
                      "secondary_scenes": all_metadata['secondary_scenes'],
-                     "sensing_start": ref_props['startTime'],
-                     "sensing_stop": ref_props['stopTime'],
-                     "orbit_number": [ref_props['orbit'], sec_props['orbit']],
-                     "platform": ref_props['platform'],
+                     "sensing_start": f'{ref_props["startTime"]}Z',
+                     "sensing_stop": f'{ref_props["stopTime"]}Z',
```

Comment:
These values will be set as the start/end times for the GUNW product in CMR and Vertex. I suspect searching for GUNW products would be more intuitive if these were the start of the earliest scene and the stop of the latest scene, rather than only the start/stop of the reference scene. I don't search for interferograms often myself, though, so it's likely worth gathering more input before making changes.

Reply:
I totally agree and thought about this myself. Unfortunately, the existing products do not do this, which means searching using these metadata fields would be inconsistent and even more annoying.

https://github.com/aria-jpl/topsApp_pge/blob/develop/create_standard_product_s1.py#L459-L478

If you have a suggestion on how to remedy this previous oversight or can propose a workflow to deal with this possible inconsistency - I am all ears.